ding/model/template/tests/test_wqmix.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import torch
+from ding.torch_utils import is_differentiable
+from ding.model.template.wqmix import MixerStar, WQMix
+#TEST
+
+@pytest.mark.unittest
+def test_mixer_star():
+    agent_num, bs, embedding_dim = 4, 3, 32
+    agent_q = torch.randn(bs, agent_num)
+    state_embedding = torch.randn(bs, embedding_dim)
+    mixer_star = MixerStar(agent_num, embedding_dim, 64)
+    total_q = mixer_star(agent_q, state_embedding)
+    assert total_q.shape == (bs, )
+    loss = total_q.mean()
+    is_differentiable(loss, mixer_star)
+
+
+@pytest.mark.unittest
+def test_wqmix():
+    is_q_star = [True, False]
```

Comment:
multi arguments in pytest can follow [this](https://github.com/opendilab/DI-engine/blob/main/ding/rl_utils/tests/test_ppo.py#L25)